src/data_utils.py

```diff
@@ -27,6 +27,57 @@ def read_tfrecords(record):
     return speech, label
 
 
+class CommonDataLoader:
+    def __init__(self, batch_size, buffer_size, audio_pad_id, labels_pad_id, audio_maxlen, labels_maxlen):
```

Comment:
Are we not using type annotations? If not then I would suggest adding docstrings everywhere. 

Ccing @MorganR for his thoughts too.

Reply:
I will add type annotations.